options/locale/locale_en-US.ini

```diff
@@ -2839,6 +2839,14 @@ settings.location = Location
 settings.permission = Permissions
 settings.repoadminchangeteam = Repository admin can add and remove access for teams
 settings.visibility = Visibility
+settings.change_visibility = Change Visibility
+settings.invalid_visibility = The new visibility is not valid.
+settings.change_visibility_notices_1 = This operation <strong>CANNOT</strong> be undone.
+settings.change_visibility_notices_2 = Some users will not visit the repositories of the orgniazation.
+settings.change_visibility_no_change = The visibility is no change.
```

Comment:
`The visibility is no change.` ?

Doesn't read like English.

Reply:
e9ba6fe6b7